app/notify/notify_email/candidate_request_confirmation.rb

```diff
@@ -60,7 +61,7 @@ def self.from_application_preview(to, application_preview)
       candidate_teaching_subject_first_choice: application_preview.teaching_subject_first_choice,
       candidate_teaching_subject_second_choice: application_preview.teaching_subject_second_choice,
       placement_outcome: application_preview.placement_outcome,
-      placement_availability: application_preview.placement_availability,
+      placement_availability: (application_preview.placement_availability || application_preview.placement_date.to_s),
```

Comment:
is it worth collapsing this `or` into a single method on `application_preview`?

Reply:
Good call, done.